docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -56,97 +60,129 @@ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+      - mysql:/var/lib/mysql
+volumes:
+  mysql:
+```
 
-If a GraphQL Playground shows up, your Prisma cluster is set up correctly.
+In the config where you see `managementApiSecret`, provide a secret you would like to use to authenticate requests to the service. In our example we are using `my-secret`.
 
-### Connect and Deploy a Service
+Now run
 
-Your Prisma cluster is secured by a public/private keypair. In order to connect with the `prisma` CLI on your local machine, you need to configure security settings.
+```sh
+docker-compose up -d
+```
 
-#### Copy cluster configuration from server
+This will fetch the docker images for both `prisma` and `mysql`. To verify that docker is running these images, run:
 
-On your digital ocean droplet, copy the clusterSecret for the `local` cluster:
+```sh
+docker ps
+```
 
 ```sh
-> cat ~/.prisma/config.yml
-clusters:
-  local:
-    host: 'http://localhost:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
+CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS                    NAMES
+24f4dd6222b1        prismagraphql/prisma:1.12   "/bin/sh -c /app/sta…"   15 seconds ago      Up 1 second         0.0.0.0:4466->4466/tcp   root_prisma_1
+d8cc3a393a9f        mysql:5.7                   "docker-entrypoint.s…"   15 seconds ago      Up 13 seconds       3306/tcp                 root_mysql_1
 ```
 
-#### Add new cluster in local configuration
+## 5. Deploy the Service
+
+Now that your infrastructure is running via docker, we will deploy the prisma service.
+
+For us to enable deploys from our local machines, open up `~/.prisma/config.yml` in your favorite editor on YOUR machine.
 
-On your local development machine open `~/.prisma/config.yml` in your favourite editor. It is likely that you already have one or more clusters defined in this file. Add another cluster and give it a descriptive name. For example `do-cluster`:
+It is likely that you already have one or more services defined in this file. Add another service and give it a descriptive name. For example do-cluster:
 
 ```yml
 clusters:
   local:
     host: 'http://localhost:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
   do-cluster:
     host: 'http://37.139.15.166:4466'
-    clusterSecret: "-----BEGIN RSA PRIVATE KEY----- [ long key omitted ] -----END RSA PRIVATE KEY-----\r\n"
 ```
 
-Change the host to use the ip of your Digital Ocean droplet.
+Next we are going to bootstrap a new project via `prisma init`. On your local machine input:
 
-#### Deploy a service
+```sh
+prisma init hello-world
+```
 
-On your local machine, verify that the cluster configuration is being picked up correctly:
+Following the interactive prompt, select `Use other server`. Enter the ip of the digital ocean droplet, and follow the prompt for adding the management secret.
```

Comment:
"Enter the ip of the digital ocean droplet, and follow the prompt for adding the management secret."

- uppercase digital ocean droplet
- should mention the prompts for service name and service stage
- I'm not aware of a prompt that's helping with the management API secret?! (in any case, the tutorial definitely also needs instructions how to set the `PRISMA_MANAGEMENT_API_SECRET` manually, without .env)

Reply:
re the prompt to add the "management API secret", which version of Prisma are you running?

Reply:
@nikolasburk using `prisma/1.12.0`. on prisma init when connecting to a remote server it prompts for managment secret